examples/astro/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "vanilla-extract-astro",
+  "description": "Example Astro project using vanilla-extract",
+  "version": "0.0.1",
+  "private": true,
+  "scripts": {
+    "start": "astro dev",
+    "build": "astro build"
+  },
+  "devDependencies": {
+    "@vanilla-extract/snowpack-plugin": "^1.1.1",
```

Comment:
You'll need to reference `1.1.0` as `1.1.1` doesn't exist yet. Don't worry though as it will be automatically updated by changesets when it releases.

Reply:
Done!